data/language/dutch.txt

```diff
@@ -2382,19 +2382,19 @@ STR_2380    :{SMALLFONT}{BLACK}Kleiner stuk water aanpassen
 STR_2381    :{SMALLFONT}{BLACK}Groter stuk water aanpassen
 STR_2382    :Land
 STR_2383    :Water
-STR_2384    :{WINDOW_COLOUR_2}Je doel:
+STR_2384    :{WINDOW_COLOUR_2}Je doelstelling:
 STR_2385    :{BLACK}Geen
-STR_2386    :{BLACK}Om minstens {COMMA16} bezoekers in je park toe hebben aan het einde van {MONTHYEAR}, met een parkwaardering van ten minste 600
-STR_2387    :{BLACK}Om een parkwaarde van minstens {POP16}{POP16}{CURRENCY} te hebben aan het einde van {PUSH16}{PUSH16}{PUSH16}{MONTHYEAR}
+STR_2386    :{BLACK}Om ten minste {COMMA16} bezoekers in je park te hebben aan het einde van {MONTHYEAR}, met een parkwaardering van ten minste 600
+STR_2387    :{BLACK}Om een parkwaarde van ten minste {POP16}{POP16}{CURRENCY} te hebben aan het einde van {PUSH16}{PUSH16}{PUSH16}{MONTHYEAR}
```

Comment:
Formeler en net wat duidelijker/stijlvoller.

Had dit overigens gedaan omdat het eerder deels ten minste of minstens was. M'n voorganger had dit blijkbaar ook gezien maar juist alles naar minstens veranderd. Dit viel me pas halverwege op. Maar aangezien ten minste m'n voorkeur hield heb ik het toch doorgezet.
Formeler en net wat duidelijker/stijlvoller.

Had dit overigens gedaan omdat het eerder deels ten minste of minstens was. M'n voorganger had dit blijkbaar ook gezien maar juist alles naar minstens veranderd. Dit viel me pas halverwege op. Maar aangezien ten minste m'n voorkeur hield heb ik het toch afgemaakt.